config/src/test/java/org/springframework/security/config/http/SessionManagementConfigTests.java

```diff
@@ -1,622 +1,635 @@
-/*
```

Comment:
Will you please double-check the diff on this file? Git thinks that you've changed the entire file when I think you only intended to add tests.

Reply:
Also, please make sure that the copyright year is updated.